### PR TITLE
FLOE-268: updated design and UIO fixes

### PR DESCRIPTION
**Pull request by jhung**
Styling changes to make the sprint page resemble the intended design.

There is a known issue if UIO is set to using Verdana and line spacing is 1.8 or higher, the text will bleed out of its containers.

### CODE REVIEW

File: accessibilitySprint2015.html [hunk 1]
```diff
@@ -4,9 +4,8 @@
         <title>OER Accessibility Sprint 2015 | floe</title>
         <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
         <link type="image/x-icon" href="images/favicon.ico" rel="shortcut icon"/>
-        <link href="css/fss/fss-reset.css" rel="stylesheet" type="text/css" />
-        <link href="css/fss/fss-layout.css" rel="stylesheet" type="text/css" />
-        <link href="css/fss/fss-text.css" rel="stylesheet" type="text/css" />
+        <link href="https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.0/css/foundation.min.css" rel="stylesheet" type="text/css" />
+        <link href="https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.0/css/normalize.min.css" rel="stylesheet" type="text/css" />
```
Comment (jobara) [inline accessibilitySprint2015.html]: I think foundation also depends on normalize, so the includes might be in the wrong order.

File: accessibilitySprint2015.html [hunk 2]
```diff
@@ -64,42 +67,76 @@
...
+                </h1>
+            </div>
+            <nav>
+                <ul class="row inline-list">
+                    <li><a href="#intro">Introduction</a></li>
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#signup">Sign-up</a></li>
+                    <!--
+                    <li><a href="#agenda">Agenda</a></li>
+                    -->
```
Comment (jobara) [inline accessibilitySprint2015.html]: can this commented out markup be removed?

**Comment by jobara** [comment]
The "Introduction" link in the header navigation doesn't seem to go anywhere.

**Comment by jobara** [comment]
Seems like we are missing links for all of the partners.

**Comment by jobara** [comment]
The keyboard focus styling for the partner icons doesn't appear around the icon.
screen shot 2015-02-19 at 11 23 22 am

File: css/sprint.css [hunk 3]
```diff
@@ -1,104 +1,267 @@
...
 nav a {
-    padding-right: 1em;
-    color: #000000;
+    margin: 0.5rem 0 0.2rem 0;
+    color: #144f4d; /* dark blue */
     text-decoration: none;
 }
 nav a:hover,
 nav a:focus {
+/*    font-weight: bold;  remove for now*/
```
Comment (jobara) [inline css/sprint.css]: can this be removed?

File: css/sprint.css [hunk 4]
```diff
@@ -1,104 +1,267 @@
...
+
+.signup-circles .yellow-circle {
+    top: -57rem;
+    left: -5rem;
+}
+
+.signup-circles p,
+.signup-circles h2 {
+    color: #fece88; /* yellow */
+
```
Comment (jobara) [inline css/sprint.css]: extra new line

File: accessibilitySprint2015.html [hunk 5]
```diff
@@ -183,48 +206,22 @@
...
-                    <li>WGBH National Center for Accessible Media</li>
-                    <li>MIT</li>
-                </ul>
-                <ul class="fl-col">
-                    <li>Raising the Floor</li>
-                    <li>George Brown College</li>
-                    <li>... and many more!</li>
-                </ul>
+            <div>
+                Over 60 global partners and institutions will be attending, RSVP today!
```
Comment (jobara) [inline accessibilitySprint2015.html]: I think this needs to be something like "Over 60 attendees from our partners across the world." Also the registration might be closed now, so we can drop the RSVP part.

**Comment by jobara** [comment]
Is there a reason to have the partner icons on two rows?

screen shot 2015-02-19 at 3 28 35 pm

**Comment by jhung** [comment]
I've added a min-width to the logo container, so now it reflows as needed.

**Comment by jobara** [comment]
merged at 106bc5e95d2beaa6fecf315538b921f8527f973f

**Pull request merged**